Shared/Tests.Sync.Shared/MergeByPKTests.cs

```diff
@@ -30,6 +30,16 @@ namespace Tests.Sync
     [TestFixture, Preserve(AllMembers = true)]
     public class MergeByPKTests
     {
+        [SetUp]
+        public void SetUp()
+        {
+            // We simulate a clean slate first time user login
```

Comment:
The problem with `ResetForTesting` is that it forces the bindings to terminate all active sessions and I'm not sure how to do that. Will logging out the user do the trick?